build-system/server/app.js

```diff
@@ -73,10 +73,7 @@ app.use(bodyParser.text());
 // Middleware is executed in order, so this must be at the top.
 // TODO(#24333): Migrate all server URL handlers to new-server/router and
 // deprecate app.js.
-// TODO(erwinmombay, #32865): Make visual diff tests use the new server
-if (!argv._.includes('visual-diff')) {
-  app.use(require('./new-server/router'));
-}
+app.use(require('./new-server/router'));
```

Comment:
Can we split the new server code into a separate PR?

Reply:
i could, but it would essentially be this single line change? unless im misunderstanding your request

Reply:
I mean, also update the server test files in that separate PR

Reply:
isolated the server changes in https://github.com/ampproject/amphtml/pull/37121/files (that needs to go in first) then this one for the visual diff changes